octree_web_viewer/src/backend.rs

```diff
@@ -30,9 +31,7 @@ impl<S> Handler<S> for VisibleNodes {
             let e: Vec<f32> = req
                 .query()
                 .get("matrix")
-                .ok_or(PointsViewerError::BadRequest(
-                    "Expected 4x4 Matrix".to_string(),
-                ))?
+                .ok_or_else(|| PointsViewerError::BadRequest("Expected 4x4 matrix".to_string()))?
```

Comment:
https://rust-lang.github.io/rust-clippy/master/#or_fun_call
This is probably not correct though, because we don't call a function, but just create an enum here...

Reply:
No, it is correct: We call `to_string()`, which does a heap allocation that is not needed on the happy path.

Reply:
Darn! Didn't scroll all the way to the right :)

Reply:
Ha! Got it, thanks @SirVer